root/images/README.md

```diff
@@ -1,6 +1,8 @@
 ## Content
 
-Images may include, but is not limited to: 
+The images section, most broadly, contains any kind of digital picture.
```

Comment:
> any kind of digital picture

IMO this is too restrictive / confusing. For some people (me included) pictures quickly becomes photo.. and I think that link is broken by just using the 'image' as a base and then branch out into pictures/photos/artworks/etc.
I think I had some resources on the exact definitions of `picture` and `photo`..

Reply:
Very true, "image" as a term is rather hard to define. I'll leave it as is until there's any better definitions available.